api/src/models/api/stats/ILatestMilestone.ts

```diff
@@ -0,0 +1,11 @@
+export interface ILatestMilestone {
+    /**
+     * The hash.
+     */
+    id?: string;
```

Comment:
Why just `id` in the name ? The later we will not be sure that this id is exactly.
If it's a milestoneId, then we should call it that.